.circleci/config.yml

```
@@ -2,6 +2,8 @@
 version: 2
 jobs:
   build:
+    machine:
+      image: ubuntu-2004:202101-01
```

unlike `machine` it works in the circleci cli
```suggestion
    docker:
      - image: cimg/base:2020.01
```